src/requestHandlers/panelSet.ts

```diff
@@ -234,68 +249,36 @@ const _getTreeController = (sequelize: Sequelize) => async(id: number) => {
     }
 };
 
-// test methods so I don't have to run all of these queries every time I want to test something
-const dumbDumb = async (request: Request, res: Response) => {
-    const response = await _dumbDumbController(sequelize)();
-    return sanitizeResponse(response, res, '');
-};
 
-const _dumbDumbController = (sequelize: Sequelize) => async() => {
-    try {
+/**
+ * Throws an error if the next_panel_set is invalid
+ * @param next_panel_set_id panel set id of the next panel set the hook is attached to
+ */
+const validateHookConnection = (sequelize : Sequelize) => async(next_panel_set_id : number, validateTrunks : boolean) =>{
```

Comment:
1. This should follow our controller naming convention
2. This should all be in a try catch just in case something goes wrong with the services
3. Throw errors is fine, but it means that nothing is being returned at the end. It might be better to return a boolean that states if it valid or not along with a message
4. I understand we have this in this file to have the unit tests pass, but I can see this being confusing later on. As this project grows I assume this issue with unit tests is going to happen a lot with other controllers. If we continue to do this, people are going to be confused on where methods are placed. I would personally like to have failing unit tests rather than changing our file structure